### PR TITLE
#251 Enable Ohara manager windows support

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #251)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by saivirtue** [comment]
Thanks a ton, @konekoya !

I will give it a shot ASAP ;)

**Comment by konekoya** [comment]
retry manager-e2e

File: ohara-manager/client/cypress/support/retryOnFail.js [hunk 1]
```diff
@@ -14,12 +14,10 @@
...
  */
 
-const integrationFolderName = Cypress.config('integrationFolder')
-  .split('/')
-  .pop();
+const isE2eTest = Cypress.config('integrationFolder').includes('e2e');
 
 // Enable cypress to retry on fail end-to-end test
-if (integrationFolderName === 'e2e') {
+if (isE2eTest === 'e2e') {
```
Comment (konekoya) [inline ohara-manager/client/cypress/support/retryOnFail.js]: Don't have a better way to get the dir name 🤷‍♂

File: ohara-manager/package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "preinstall": "node ./scripts/checkYarnVersion.js",
     "setup": "yarn && yarn --cwd client setup",
-    "start": "nodemon start.js --exec 'yarn lint:server && node'",
+    "start": "nodemon start.js --exec \"yarn lint:server && node\"",
```
Comment (konekoya) [inline ohara-manager/package.json]: Windows doesn't like single quote inside double quote...

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**